Src/main.c

```diff
@@ -72,6 +72,12 @@ int main(void)
 
   /* USER CODE BEGIN 1 */
 
+    // Variables. 
+    unsigned short potentiometerValue;
```

Comment:
I'd rather you use uint16_t here instead :))

https://pubs.opengroup.org/onlinepubs/009604599/basedefs/stdint.h.html